lib/OpenQA/Worker/Job.pm

```diff
@@ -439,9 +441,9 @@ sub _stop_step_5_upload {
         }
 
         # upload other logs and files
-        for my $file (
-            qw(video.ogv video_time.vtt vars.json serial0 autoinst-log.txt serial_terminal.txt virtio_console.log worker-log.txt)
-          )
+        my @files_to_upload = split(m/,/, $self->info->{settings}->{AUTOINST_UPLOAD_FROM_POOL});
+        @files_to_upload = UPLOAD_FROM_POOL unless @files_to_upload;
```

Comment:
This would also not allow you to disable all files to upload with a setting link `AUTOINST_UPLOAD_FROM_POOL=''`

Reply:
didn't get your point - you think we should allow this ?

Reply:
Yes, for me it is consistence. If I set `AUTOINST_UPLOAD_FROM_POOL` to what ever, what ever is used.

Reply:
> Yes, for me it is consistence. If I set `AUTOINST_UPLOAD_FROM_POOL` to what ever, what ever is used.

are you sure ? 
Just test it with code : 
```
my $settings = ' ';
my @files_to_upload = split m/,/, $settings;
@files_to_upload = qw(w e r) unless @files_to_upload;
print "x=(@files_to_upload)";
# Output : x=( ) 
```